compiler/damlc/tests/daml-test-files/RetroactiveInterfaceInstance.EXPECTED.rst

```diff
@@ -0,0 +1,131 @@
+.. _module-retroactiveinterfaceinstance-60009:
+
+Module RetroactiveInterfaceInstance
+-----------------------------------
+
+Interfaces
+^^^^^^^^^^
+
+.. _type-retroactiveinterfaceinstance-token-49693:
+
+**interface** `Token <type-retroactiveinterfaceinstance-token-49693_>`_
+
+  **viewtype** `TokenView <type-retroactiveinterfaceinstance-tokenview-57374_>`_
+  
+  + **Choice GetRich**
+    
+    .. list-table::
+       :widths: 15 10 30
+       :header-rows: 1
+    
+       * - Field
+         - Type
+         - Description
+       * - byHowMuch
+         - `Int <https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-37261>`_
+         - 
+  
+  + **Choice Noop**
+    
+    .. list-table::
+       :widths: 15 10 30
+       :header-rows: 1
+    
+       * - Field
+         - Type
+         - Description
+       * - nothing
+         - ()
+         - 
+  
+  + **Choice Split**
+    
+    An interface choice comment\.
+    
+    .. list-table::
+       :widths: 15 10 30
+       :header-rows: 1
+    
+       * - Field
+         - Type
+         - Description
+       * - splitAmount
+         - `Int <https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-37261>`_
+         - A choice field comment\.
+  
+  + **Choice Transfer**
+    
+    .. list-table::
+       :widths: 15 10 30
+       :header-rows: 1
+    
+       * - Field
+         - Type
+         - Description
+       * - newOwner
+         - `Party <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-party-57932>`_
+         - 
+  
+  + **Method noopImpl \:** () \-\> `Update <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-68072>`_ ()
+  
+  + **Method setAmount \:** `Int <https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-37261>`_ \-\> `Token <type-retroactiveinterfaceinstance-token-49693_>`_
+  
+  + **Method splitImpl \:** `Int <https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-37261>`_ \-\> `Update <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-68072>`_ (`ContractId <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-95282>`_ `Token <type-retroactiveinterfaceinstance-token-49693_>`_, `ContractId <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-95282>`_ `Token <type-retroactiveinterfaceinstance-token-49693_>`_)
+  
+  + **Method transferImpl \:** `Party <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-party-57932>`_ \-\> `Update <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-68072>`_ (`ContractId <https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-95282>`_ `Token <type-retroactiveinterfaceinstance-token-49693_>`_)
+
+  + **interface instance** `Token <type-retroactiveinterfaceinstance-token-49693_>`_ **for** Asset
```

Comment:
there's no hyperlink on `Asset` but that's because these golden tests are generated for one `.daml` file at a time, so there's nothing to link to from here. The new tests in `compiler/damlc/daml-doc/test/DA/Daml/Doc/Tests.hs` check that the links are there if both files are supplied to daml-doc.